pyfpga/project.py

```diff
@@ -110,7 +110,7 @@ def add_vhdl(self, pathname, lib=None):
         :type lib: str, optional
         :raises FileNotFoundError: when pathname is not found
         """
-        self.logger.debug('Executing add_vhdl')
+        self.logger.debug(f'Executing add_vhdl: {"default library" if lib is None else lib} : {pathname}')
```

Comment:
This line has more than 79 characters (pycodestyle/pylint complains). Maybe you could compute library before. And could you use `'` instead of `"`? Thanks.